drivers/vulkan/vulkan_context.cpp

```diff
@@ -1361,7 +1362,88 @@ Error VulkanContext::_create_physical_device(VkSurfaceKHR p_surface) {
 		return ERR_CANT_CREATE;
 	}
 
-	physical_device_features.robustBufferAccess = false; // Turn off robust buffer access, which can hamper performance on some hardware.
+	memset(&physical_device_features, 0, sizeof(physical_device_features));
+#define VK_DEVICEFEATURE_ENABLE_IF(x) \
+	if (features.x)                   \
+	physical_device_features.x = features.x
```

Comment:
We try to respect the clang-format code style even in macro code, so this would be:
```suggestion
	if (features.x) {                  \
		physical_device_features.x = features.x \
	}
```
(with the trailing backslash aligned like clang-format wants it, didn't check)

Reply:
The macro respects clang-format.

As for adding braces, we have a small problem. The following:

```cpp
#define VK_DEVICEFEATURE_ENABLE_IF(x) \
	if (features.x) {                  \
	physical_device_features.x = features.x; }
VK_DEVICEFEATURE_ENABLE_IF(fullDrawIndexUint32); // <- Warning
```

Generates a warning because it has a stray `;`

> vulkan_context.cpp:1403:44: warning: empty expression statement has no effect; remove unnecessary ';' to silence this warning

There are ugly workarounds to that but I think we can let this macro slide and add a comment explaining why it is an exception?

Reply:
Oh right, we usually use
```
} else
	((void)0)
```
as a workaround (see e.g. `core/error/error_macros.h`)

But yeah we can make an exception here, it's not critical. Maybe just ensure the generated code still has proper indentation:
```suggestion
	if (features.x)                   \
		physical_device_features.x = features.x
```

Reply:
It _is_ Clang Format who removes the indentation.

Anyway, I'll go with:

```cpp
#define VK_DEVICEFEATURE_ENABLE_IF(x)           \
	if (features.x) {                           \
		physical_device_features.x = features.x \
	} else                                      \
		((void)0)
```

which makes Clang Format indent correctly.